test/jdk/java/foreign/libTestDowncall.h

```diff
@@ -1,12210 +0,0 @@
-struct S_IFI { int p0; float p1; int p2; };
-struct S_IFF { int p0; float p1; float p2; };
-struct S_IFD { int p0; float p1; double p2; };
-struct S_IFP { int p0; float p1; void* p2; };
-struct S_IDI { int p0; double p1; int p2; };
-struct S_IDF { int p0; double p1; float p2; };
-struct S_IDD { int p0; double p1; double p2; };
-struct S_IDP { int p0; double p1; void* p2; };
-struct S_IPI { int p0; void* p1; int p2; };
-struct S_IPF { int p0; void* p1; float p2; };
-struct S_IPD { int p0; void* p1; double p2; };
-struct S_IPP { int p0; void* p1; void* p2; };
-struct S_FII { float p0; int p1; int p2; };
-struct S_FIF { float p0; int p1; float p2; };
-struct S_FID { float p0; int p1; double p2; };
-struct S_FIP { float p0; int p1; void* p2; };
-struct S_FFI { float p0; float p1; int p2; };
-struct S_FFF { float p0; float p1; float p2; };
-struct S_FFD { float p0; float p1; double p2; };
-struct S_FFP { float p0; float p1; void* p2; };
-struct S_FDI { float p0; double p1; int p2; };
-struct S_FDF { float p0; double p1; float p2; };
-struct S_FDD { float p0; double p1; double p2; };
-struct S_FDP { float p0; double p1; void* p2; };
-struct S_FPI { float p0; void* p1; int p2; };
-struct S_FPF { float p0; void* p1; float p2; };
-struct S_FPD { float p0; void* p1; double p2; };
-struct S_FPP { float p0; void* p1; void* p2; };
-struct S_DII { double p0; int p1; int p2; };
-struct S_DIF { double p0; int p1; float p2; };
-struct S_DID { double p0; int p1; double p2; };
-struct S_DIP { double p0; int p1; void* p2; };
-struct S_DFI { double p0; float p1; int p2; };
-struct S_DFF { double p0; float p1; float p2; };
-struct S_DFD { double p0; float p1; double p2; };
-struct S_DFP { double p0; float p1; void* p2; };
-struct S_DDI { double p0; double p1; int p2; };
-struct S_DDF { double p0; double p1; float p2; };
-struct S_DDD { double p0; double p1; double p2; };
-struct S_DDP { double p0; double p1; void* p2; };
-struct S_DPI { double p0; void* p1; int p2; };
-struct S_DPF { double p0; void* p1; float p2; };
-struct S_DPD { double p0; void* p1; double p2; };
-struct S_DPP { double p0; void* p1; void* p2; };
-struct S_PII { void* p0; int p1; int p2; };
-struct S_PIF { void* p0; int p1; float p2; };
-struct S_PID { void* p0; int p1; double p2; };
-struct S_PIP { void* p0; int p1; void* p2; };
-struct S_PFI { void* p0; float p1; int p2; };
-struct S_PFF { void* p0; float p1; float p2; };
-struct S_PFD { void* p0; float p1; double p2; };
-struct S_PFP { void* p0; float p1; void* p2; };
-struct S_PDI { void* p0; double p1; int p2; };
-struct S_PDF { void* p0; double p1; float p2; };
-struct S_PDD { void* p0; double p1; double p2; };
-struct S_PDP { void* p0; double p1; void* p2; };
-struct S_PPI { void* p0; void* p1; int p2; };
-struct S_PPF { void* p0; void* p1; float p2; };
-struct S_PPD { void* p0; void* p1; double p2; };
-struct S_PPP { void* p0; void* p1; void* p2; };
-EXPORT void f0_V__(void) ;
-EXPORT void f0_V_I_(int p0) ;
```

Comment:
I assume you removed the functions from the header because they are not needed - e.g. the definition in the C file is sufficient?

Reply:
Right, only the struct definitions are needed